utils/reviewcomment.go

```diff
@@ -53,8 +112,8 @@ func AddReviewComments(repo *Repository, pullRequestID int, client vcsclient.Vcs
 	return
 }
 
-func deleteOldReviewComments(repo *Repository, pullRequestID int, client vcsclient.VcsClient) (err error) {
-	// Get all comments in PR
+func deleteExistingPullRequestReviewComments(repo *Repository, pullRequestID int, client vcsclient.VcsClient) (err error) {
```

Comment:
Let's add a unit test for this function.

Reply:
done